mend_import_sbom/import_sbom.py

```diff
@@ -658,8 +672,16 @@ def main():
             hdr_title = f'{APP_TITLE} {AGENT_INFO["agentVersion"]}'
             hdr = f'\n{len(hdr_title) * "="}\n{hdr_title}\n{len(hdr_title) * "="}'
             logger.info(hdr)
-
             log_obj_props(args, "Configuration:")
+            data = json.dumps(
+                {
+                    "requestType": "getOrganizationDetails",
+                    "orgToken": args.ws_token ,
+                    "userKey": args.ws_user_key
+            }
```

Comment:
Agent info needs to be added to this call.

Reply:
Added into call_api function